[2025-01-29T00:49:58.338+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T00:49:58.355+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-01-29T00:49:55.860332+00:00 [queued]>
[2025-01-29T00:49:58.364+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-01-29T00:49:55.860332+00:00 [queued]>
[2025-01-29T00:49:58.365+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-29T00:49:58.381+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-01-29 00:49:55.860332+00:00
[2025-01-29T00:49:58.388+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1080) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T00:49:58.389+0000] {standard_task_runner.py:72} INFO - Started process 1083 to run task
[2025-01-29T00:49:58.389+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'manual__2025-01-29T00:49:55.860332+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpjtfhluvd']
[2025-01-29T00:49:58.391+0000] {standard_task_runner.py:105} INFO - Job 42: Subtask check_directory
[2025-01-29T00:49:58.704+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-01-29T00:49:55.860332+00:00 [running]> on host f85a165ecee3
[2025-01-29T00:49:58.818+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T00:49:55.860332+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T00:49:55.860332+00:00'
[2025-01-29T00:49:58.819+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-29T00:49:58.820+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-29T00:49:58.820+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-01-29 00:49:58.356238+00:00
[2025-01-29T00:49:58.821+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-01-29T00:49:58.822+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T00:49:58.823+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-29T00:49:58.824+0000] {logging_mixin.py:190} INFO - 1boris True
[2025-01-29T00:49:58.825+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-01-29T00:49:58.826+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-01-29T00:49:58.827+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-01-29T00:49:58.841+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-01-29T00:49:58.874+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T00:49:58.875+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=manual__2025-01-29T00:49:55.860332+00:00, execution_date=20250129T004955, start_date=20250129T004958, end_date=20250129T004958
[2025-01-29T00:49:58.888+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-29T00:49:58.888+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-29T00:49:58.889+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-01-29 00:49:55.869999+00:00
[2025-01-29T00:49:58.890+0000] {logging_mixin.py:190} INFO - Task hostname:f85a165ecee3 operator:BranchPythonOperator
[2025-01-29T00:49:58.926+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T00:49:59.153+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T00:49:59.155+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
